test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -709,6 +709,23 @@ public final void createIndex(String... names) {
     }
 
     /**
+     * Creates a shadow replica index and asserts that the index creation was acknowledged.
+     * Can only be invoked on a cluster where each node has been configured with shared data
+     * paths and the other necessary settings for shadow replicas.
+     */
+    public final void createShadowReplicaIndex(final String name, final Path dataPath, final int numReplicas) {
```

Comment:
The reason I put it there is because that's typically where one would go to see if such a function is already available to them and if not, potentially recreate one in their own test without knowing it exists in another IT test.  I thought this was one of those "generally useful" functions that could be used elsewhere, but if its not, I can move it out.